.github/CHANGELOG.md

```diff
@@ -439,31 +374,74 @@ K_test = qml.kernels.kernel_matrix(X_train, X_test, kernel)
+  - The IsingZZ gate `qml.IsingZZ` [(#1199)](https://github.com/PennyLaneAI/pennylane/pull/1199)
+  - The ISWAP gate `qml.ISWAP` [(#1298)](https://github.com/PennyLaneAI/pennylane/pull/1298)
+  - The reset error noise channel `qml.ResetError` [(#1321)](https://github.com/PennyLaneAI/pennylane/pull/1321)
 
-* Ising ZZ gate functionality added.
-  [(#1199)](https://github.com/PennyLaneAI/pennylane/pull/1199)
 
-* The ISWAP operation has been added to the `default_qubit` device.
-  [(#1298)](https://github.com/PennyLaneAI/pennylane/pull/1298)
+<h3>Improvements</h3>
 
-* Ising XX gate functionality added. [(#1194)](https://github.com/PennyLaneAI/pennylane/pull/1194)
+* The ``argnum`` keyword argument can now be specified for a QNode to define a
+  subset of trainable parameters used to estimate the Jacobian.
+  [(#1371)](https://github.com/PennyLaneAI/pennylane/pull/1371)
 
-* Added Projector observable, which is available on all devices inheriting from the `QubitDevice` class.
-  [(#1356)](https://github.com/PennyLaneAI/pennylane/pull/1356)
+  For example, consider two trainable parameters and a quantum function:
 
-<h3>Improvements</h3>
+  ```python
+  dev = qml.device("default.qubit", wires=2)
+
+  x = np.array(0.543, requires_grad=True)
+  y = np.array(-0.654, requires_grad=True)
+
+  def circuit(x,y):
+      qml.RX(x, wires=[0])
+      qml.RY(y, wires=[1])
+      qml.CNOT(wires=[0, 1])
+      return qml.expval(qml.PauliZ(0) @ qml.PauliX(1))
+  ```
+
+  When computing the gradient of the QNode, we can specify the trainable
+  parameters to consider by passing the ``argnum`` keyword argument:
+
+  ```pycon
+  >>> qnode1 = qml.QNode(circuit, dev, diff_method="parameter-shift", argnum=[0,1])
+  >>> print(qml.grad(qnode1)(x,y))
+  (array(0.31434679), array(0.67949903))
+  ```
+
+  Specifying a proper subset of the trainable parameters will estimate the
+  Jacobian:
+
+  ```pycon
+  >>> qnode2 = qml.QNode(circuit, dev, diff_method="parameter-shift", argnum=[0])
+  >>> print(qml.grad(qnode2)(x,y))
+  (array(0.31434679), array(0.))
+  ```
+
+* Allows creating differentiable observables that return custom objects such
+  that the observable is supported by devices.
+  [(1291)](https://github.com/PennyLaneAI/pennylane/pull/1291)
+
+* PennyLane NumPy now includes the
+  [random module's](https://numpy.org/doc/stable/reference/random/index.html#module-numpy.random)
+  `Generator` objects, the recommended way of random number generation. This allows for
+  random number generation using a local, rather than global seed.
+  [(#1267)](https://github.com/PennyLaneAI/pennylane/pull/1267)
+
+  ```python
+  from pennylane import numpy as np
+
+  rng = np.random.default_rng()
+  random_mat1 = rng.random((3,2))
+  random_mat2 = rng.standard_normal(3, requires_grad=False)
+  ```
 
 * The adjoint jacobian differentiation method reuses the state computed on the forward pass.
   This can be turned off to save memory with the Torch and TensorFlow interfaces by passing
```

Comment:
It might be good to say what the improvement is here. In this case, it leads to a significant performance improvement.